modules/adfinityBidAdapter.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+```
+Module Name: Adfinity Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: adfinity_prebid@i.ua
+```
+
+# Description
+
+Module that connects to Adfinity demand sources
+
+# Test Parameters
+```
+    var adUnits = [{
+                code: 'placementid_0',
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250], [300,600]]
+                    }
+                },
+                bids: [{
+                        bidder: 'adfinity',
+                        params: {
+                            placement_id: 0,
+                            traffic: 'banner'
+                        }
+                    },
+                    {
+                        bidder: 'afinity',
+                        params: {
+                            placement_id: 0,
+                            traffic: 'video'
+                        }
+                    },
+                    {
+                        bidder: 'afinity',
+                        params: {
+                            placement_id: 0,
+                            traffic: 'native'
+                        }
+                    }
```

Comment:
Can you restructure these test params into separate adUnits with their own `mediaTypes`?  I assume you're supporting `instream` video only?  I can't really confirm.

The same applies to `native`, in terms of which assets you support.